docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,107 @@
+level, sex, sexual orientation, gender identity and expression, age,
+size, family status, political belief, religion or lack thereof, and
+mental and physical ability.
+
+- Be considerate. Your work will be used by other people, and you in
+turn will depend on the work of others. Any decision you take will
+affect users and colleagues, and you should take those consequences
+into account. Remember that we are a world-wide community, so you
+might not be communicating in someone else’s primary language.
+
+- Be respectful. Not all of us will agree all the time, but
+disagreement is no excuse for poor behavior and poor manners. We might
+all experience some frustration now and then, but we cannot allow that
+frustration to turn into a personal attack. It is important to
+remember that a community where people feel uncomfortable or
+threatened is not a productive one. Members of the SpECTRE community
+should be respectful when dealing with other members as well as with
+people outside the community.
+
+- Be careful in the words that you choose and be kind to others. Do
+not insult or put down other participants. Harassment and other
+exclusionary behavior aren’t acceptable. This includes, but is not
+limited to:
+  - Violent threats or language directed against another person.
+  - Discriminatory jokes and language.
+  - Posting sexually explicit or violent material.
+  - Posting (or threatening to post) other people’s personally
+  identifying information (“doxing”).
+  - Personal insults, especially those using racist or sexist terms.
+  - Unwelcome sexual attention.
+  - Advocating for, or encouraging, any of the above behavior.  In
+general, if someone asks you to stop, then stop. Persisting in such
+behavior after being asked to stop is considered harassment.
+
+- When we disagree, try to understand why. Disagreements, both social
+and technical, happen all the time, and SpECTRE is no exception. It is
+important that we resolve disagreements and differing views
+constructively. Remember that we are different. The strength of
+communities comes from having a varied community, people from a wide
+range of backgrounds. Different people have different perspectives on
+issues. Being unable to understand why someone holds a viewpoint does
+not mean that they are wrong. Do not forget that it is human to err
+and blaming each other does not get us anywhere. Instead, focus on
+helping to resolve issues and learning from mistakes.
+
+### Attribution
+
+This Code of Conduct is adapted from the [SXS Collaboration Code of
+Conduct](https://www.black-holes.org/sxs_code_of_conduct.pdf), which
+is based on a draft of the [LLVM Code of
+Conduct](https://llvm.org/docs/CodeOfConduct.html) which is adapted
+from [Django Project Code of
+Conduct](https://www.djangoproject.com/conduct/) which used text from
+the [Speak Up! Code of
+Conduct](http://web.archive.org/web/20141109123859/http://speakup.io/coc.html).
```

Comment:
I'm very happy that we are adding the code of conduct to the spectre GitHub! I suggest mentioning here as well how to report potential violations.

Reply:
It is mentioned on lines 35-37.